site/src/components/ParameterInput/ParameterInput.tsx

```diff
@@ -102,13 +101,13 @@ const ParameterField: React.FC<ParameterInputProps> = ({
         <FormControlLabel
           disabled={disabled}
           value="true"
-          control={<Radio color="primary" size="small" disableRipple />}
+          control={<Radio color="primary" disableRipple />}
```

Comment:
I see we're still using `color="primary"` in a few places - seems like maybe we can get rid of this since it's now [the default](https://mui.com/material-ui/migration/v5-component-changes/#%E2%9C%85-remove-default-color-prop)?